templates/components/footer.html.hbs

```diff
@@ -31,6 +31,7 @@
       <div class="flex flex-column mw8 w-100 measure-wide-l pv2 pv5-m pv2-ns ph4-m ph4-l">
         <h4>{{fluent "footer-social"}}</h4>
         <div class="flex flex-row flex-wrap items-center">
+          <a rel="me" href="https://social.rust-lang.org/@rust" target="_blank"><img src="/static/images/mastodon.svg" alt="mastodon logo" title="Mastodon" /></a>
```

Comment:
Please add Fluent keys for this. It's fine to call it Mastodon in each langauge for now